docs/upgrading.md

```diff
@@ -29,6 +31,29 @@
     - [Removing unnecessary metadata using the resourceprocessor](#removing-unnecessary-metadata-using-the-resourceprocessor)
     - [Moving record-level attributes used for metadata to the resource level](#moving-record-level-attributes-used-for-metadata-to-the-resource-level)
 
+## Upgrading to v0.84.0-sumo-0
+
+### `sumologic` extension: removed `install_token` in favor of `installation_token`
+
+The `install_token` configuration property was deprecated in `v0.72.0-sumo-0` in February 2023 in favor of `installation_token`.
+It is now being removed in `v0.84.0-sumo-0`. To upgrade, replace `install_token` property with `installation_token` in your Sumo Logic extension configuration.
+
+For example, change this:
+
+```yaml
+extensions:
+  sumologic:
+    install_token: xyz
+```
+
+to this:
+
+```yaml
+extensions:
+  sumologic:
+    installation_token: xyz
+```
+
```

Comment:
Why wasn't this doc entry added with the change?

Reply:
I believe it should have been, but it was missed. https://github.com/SumoLogic/sumologic-otel-collector/pull/1225